### PR TITLE
Refactor/error handling

### LINKED ISSUE #32: Update README.md with New Content
After iteration 2, update readme as needed with new images/gifs and info.

### LINKED ISSUE #38: Final Refactor - part 2
"There are some spots where the indentation is funky - get an auto formatter so you don’t have to think about it!" - Jeremiah

### PR DESCRIPTION
**Pull request by King13k**
## Summary:
Refactored the code making sure we had no console.logs hanging around and updated the readMe file.

## Type of Changes:
- [ ] Bug fix
- [x] Refactor
- [ ] New feature

## Where are the Changes:
README.md
Scripts.js
Styles.css
apiCalls.js

## How was this tested:
tested in the browser

## What are the relevant tickets:
closes #38 
closes #32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,14 +39,19 @@ The details of this project are outlined in the <a href="https://frontend.turing
...
 ## Challenges:
 - Search bar functionality of this application was difficult to implement due to the fact it had to be able to search by name or tag.
 - Initial setup with class structure
+- Pantry class integration
+- Refactoring the code to separate the DOM manipulation into its own file  
 
 ## Wins:
 - UX/UI layout
 - Our initial planning and design layout set us up for success.
+- Team work
```
Comment (kendallm360) [inline README.md:54]: Yes, good job here fellas.

File: src/apiCalls.js [hunk 2]
```diff
@@ -1,19 +1,34 @@
 const fetchData = (extension) => {
   return fetch(`http://localhost:3001/api/v1/${extension}`)
-    .then(response => response.json())
+    .then((response) => {
```
Comment (kendallm360) [inline src/apiCalls.js:3]: Nice implementation of error handling

**Comment by JoeRupp** [comment]
Amazing work team!

File: src/apiCalls.js [hunk 3]
```diff
@@ -1,19 +1,34 @@
...
+      body: JSON.stringify(data),
+      headers: {
+        "Content-Type": "application/json"
+      }
+    })
+    .then((response) => {
+      if (response.ok) {
+        return response.json();
+      } else {
+        throw Error(response.statusText)
```
Comment (JoeRupp) [inline src/apiCalls.js:25]: Nice work adding the error handling

**Pull request merged**